docs/sdks/uid2-sdk-ref-python.md

```diff
@@ -192,48 +242,48 @@ If you're using server-only integration (see [Publisher Integration Guide, Serve
 
 ## Usage for UID2 Sharers
 
-A UID2 sharer is any participant that wants to share UID2s with another participant. Raw UID2s must be encrypted into UID2 tokens before sending them to another participant. For an example of usage, see [examples/sample_sharing.py](https://github.com/IABTechLab/uid2-client-python/blob/master/examples/sample_sharing.py) script.
+A UID2 sharer is any participant that wants to share UID2s with another participant. Raw UID2s must be encrypted into UID2 tokens before sending them to another participant.
 
->IMPORTANT: The UID2 token generated during this process is for sharing only&#8212;you cannot use it in the bid stream. There is a different workflow for generating tokens for the bid stream: see [Sharing in the Bid Stream](../sharing/sharing-bid-stream.md).
+:::warning
+The UID2 token generated during this process is for sharing only&#8212;you cannot use it in the bid stream. There is a different workflow for generating tokens for the bid stream: see [Sharing in the Bid Stream](../sharing/sharing-bid-stream.md).
+:::
 
 The following instructions provide an example of how you can implement sharing using the UID2 SDK for Python, either as a sender or a receiver.
 
-1. Create a `UID2Client` reference:
- 
-   ```py
-   from uid2_client import Uid2ClientFactory
-   client = Uid2ClientFactory.create(base_url, auth_key, secret_key)
-   ```
+1. Create a `SharingClient`:
+
+```py
+client = SharingClient(UID2_BASE_URL, UID2_API_KEY, UID2_SECRET_KEY)
+```
 
 2. Refresh once at startup, and then periodically (recommended refresh interval is hourly):
 
-   ```py
-   client.refresh_keys()
-   ```
+```py
+client.refresh()
+```
 
-3. Senders: 
-   1. Call the `encrypt` function. Then, if encryption succeeded, send the UID2 token to the receiver:
+3. If you are a sender, call `encrypt_raw_uid_into_token()`:
 
-      ```py
-      try:
-         encrypted_data = client.encrypt(raw_uid)
-         # send encrypted_data to receiver
-      except EncryptionError as err:
-        # check for failure reason
-        print(err)
-      ```
+```py
+encrypted = client.encrypt_raw_uid_into_token(raw_uid)
+# If encryption succeeded, send the UID2 token to the receiver.
+if encrypted.success:
+    # Send encrypted.uid to receiver
```

Comment:
In java, we send the `encrypted.getEncryptedData()` not uid, is it the same as `encrypted.uid` here?

Reply:
you are right, it should be `encrypted.encrypted_data`. Thanks for catching this